llvm-15.0.3/llvm-crash-analyzer/lib/Analysis/TaintDataFlowGraph.cpp

```diff
@@ -134,13 +134,19 @@ void TaintDataFlowGraph::findBlameFunction(Node *v) {
               if (a->MI->getParent() == adjNode->MI->getParent() &&
                   !a->CallMI && !adjNode->CallMI) {
                 if (MDT->dominates(adjNode->MI, a->MI)) {
+                  // Do not erase potential blame nodes.
```

Comment:
why didn't you terminate in the begining of the loop (line 134) after?
the terminating condition is repeated all over this loop which makes it hard to understand